conf/namespace/test_namespace.yaml

```diff
@@ -1,12 +1,8 @@
 namespace: "test_namespace"
 frontend:
   security:
-    cert: "./ttt"
```

Comment:
Now that we have a `default.yaml`, I don't think we need `test_namespace.yaml` anymore.